Dockerfile

```diff
@@ -18,6 +18,7 @@ FROM php:7.4-fpm-alpine
 
 WORKDIR /var/www/html
 
+COPY --from=vendor /usr/bin/composer /usr/bin/composer
```

Comment:
@jorgemuza Thanks for the PR 😄

The `composer` executable was intentionally not copied in the image. The image is intended to be used as a production image and don't require `composer`. We can use `make composer` to start an `sh` session with `composer` command available.

If there is a need for `composer` command to be available in `prod` environment then it can be added additionally and not copied by default.

I would suggest to either not copy it in the image or at least comment it out and add a note above so it can be used if necessary. Eg.

```
# Notes about not using it in production...
# COPY --from=vendor /usr/bin/composer /usr/bin/composer
```